### PR TITLE
Allow mouse dragging around anything drawable (unless class blocks it specifically)

### PR DESCRIPTION
**Pull request by magnificus**
more polish to be added, but pretty happy with this conceptually

### CODE REVIEW

File: src/nodes/draw/abstract.tsx [hunk 1]
```diff
@@ -127,8 +137,46 @@ export abstract class DRAW_Base extends PPNode {
...
+      this.setOffsets(
+        new PIXI.Point(originalOffsets.x + diffX, originalOffsets.y + diffY)
+      );
+      // we dont re-trigger immediately, to save perfomance
+      setTimeout(
+        () => this.pointerDown(originalCursorPos, originalOffsets),
+        16
+      );
+    });
+    this.deferredGraphics.on('pointerup', () => {
```
Comment (fakob) [inline src/nodes/draw/abstract.tsx:164]: Maybe it is my touchpad, but sometimes when I release (pointerup) the shape continues moving. Then I have to click and release a second time.

**Comment by fakob** [comment]
Nice work

**Review by fakob — APPROVED** [review]
I saw that you can setOffsets by clickin on the drawing and move it. 
Not sure I understood what inputAlwaysDraw does.

File: src/nodes/draw/abstract.tsx (hunk 1 above)
Comment (magnificus) [reply src/nodes/draw/abstract.tsx:164]: Yes its a bug ATM because it only listens to up events on itself, going to add a new listener to address this

**Pull request merged**